shelley/chain-and-ledger/executable-spec/test/Rules/TestDeleg.hs

```diff
@@ -56,18 +60,25 @@ traceLen = 100
 -- | Check that a newly registered key has a reward of 0.
 rewardZeroAfterReg :: Property
 rewardZeroAfterReg = withTests (fromIntegral numberOfTests) . property $ do
-  t <- forAll (trace @DELEG $ fromIntegral traceLen)
+  t <- forAll (trace @LEDGER $ fromIntegral traceLen)
```

Comment:
mauybe keep this here and add a module `TestLedger` with the adapted property?

Reply:
good idea yes, see `TestLedger.hs`